tilavarauspalvelu/api/graphql/types/application_round/filtersets.py

```diff
@@ -32,6 +34,14 @@ def filter_by_active(self, queryset: models.QuerySet, name: str, value: bool) ->
             )
         )
 
+    def filter_by_ongoing(self, queryset: models.QuerySet, name: str, value: bool) -> models.QuerySet:
+        return queryset.filter(
+            models.Q(
+                L(status=ApplicationRoundStatusChoice.RESULTS_SENT),
```

Comment:
should be exclusion of this state, everything else before it is reached is considered still ongoing

Reply:
Oops, good catch!